frontend/src/screens/learner.tsx

```diff
@@ -71,57 +70,99 @@ const CuratedStudies: FC = () => {
 
 const LearnerDashboard = () => {
     const env = useEnvironment()
+    const [scroll, scrollTo] = useWindowScroll();
 
     if (!env.isEligible) {
-        return <UnsupportedCountryModal />
+        return <UnsupportedCountryModal />;
     }
+    const isVisible = scroll.y > window.innerHeight * 0.25;
 
     return (
-        <div className="studies learner">
+        <div className='studies learner'>
             <Routes>
                 <Route path={'details/:studyId'} element={<StudyDetails />} />
             </Routes>
 
             <TopNavBar />
 
             <LearnerWelcomeModal />
-
             {/* Temporarily removing this as well until reward system reworked */}
-            {/*<RewardsProgressBar />*/}
+            {/* {/<RewardsProgressBar />/} */}
 
             {/* Temporarily disable syllabus contest due to legal, keep it just in case we re-enable in the future */}
-            {/*<SyllabusContest studies={syllabusContestStudies} />*/}
+            {/* {/<SyllabusContest studies={syllabusContestStudies} />/} */}
 
             <HighlightedStudies />
 
             <StudiesContainer />
-
+            <Affix position={{ bottom: 20, left: 20 }}>
```

Comment:
This entire component can be simplied into 
`<Affix position={{ bottom: 20, left: 20 }}>
                <IconCircleArrowUpFilled onClick={() => {window.scroll(0, 0)}} size='1.5rem' style={{ color: colors.purple }}/>
</Affix>`. visibility still needs to be handled inside a useEffect hook.